transformers/senml/transformer.go

```diff
@@ -5,10 +5,16 @@ package senml
 
 import (
 	"github.com/mainflux/mainflux/broker"
+	"github.com/mainflux/mainflux/errors"
 	"github.com/mainflux/mainflux/transformers"
 	"github.com/mainflux/senml"
 )
 
+var (
+	errDecode    = errors.New("failed to decode SenML")
+	errNormalize = errors.New("faled to normalize SenML")
```

Comment:
Do we want capital letters in error messages?

Reply:
No, we don't. Thanks